### PR TITLE
End of service feedback component

### PR DESCRIPTION
**Pull request by smcmurtry**
Part of https://github.com/cds-snc/simplify-privacy-statements-V2/issues/141 / https://trello.com/c/beA8Z11P/91-update-the-feedback-on-the-generator-to-reflect-the-cds-feedback-component

This PR adds the "end of service feedback". It still needs a bit of translation. Still to come: the mid-service feedback component.

### CODE REVIEW

File: routes/feedback/feedback.njk [hunk 1]
```diff
@@ -1,21 +1,15 @@
 {% extends "base.njk" %}
 
 {% block content %}
+    <a href="agreement-1">&#9666; {{  __("back") }}</a>
+    <h1>{{ __("post-service-feedback.title") }}</h1>
 
-    <h1>{{ __('feedback.title') }}</h1>
-
-    <div>
-    {% if notifySetup %}
```
Comment (sastels) [inline routes/feedback/feedback.njk]: why take out the `{% if notifySetup %}`?

Comment (smcmurtry) [reply routes/feedback/feedback.njk]: Now the form data is sent to the feedback collector:
`action="https://cds-feedback-collector.herokuapp.com/en/send?id=c966af51-e0a3-412f-99fd-47411788da9f"`

and that sends an email through notify if you've configured it to there.

**Comment by keithiopia** [comment]
Sorry I've not had a chance to look at this yet – will do so on Monday though!

File: routes/feedback/feedback.njk (hunk 1 above)
Comment (sastels) [reply routes/feedback/feedback.njk]: nice.

**Comment by smcmurtry** [comment]
@keithiopia That is indeed a bug! I just filed it here: #144. That exists on the live site though so it was not introduced as part of this PR: https://privacy-statements.cds.alpha.canada.ca/en/

Re: the link being too prominent: the intention we had when developing the feedback component was to collect feedback at 2 separate times: 

1. during the service, in case the user has a problem that is preventing them from using the service: (mid-service example](https://feedback-plugin-example.herokuapp.com/en/personal)
2. at the end of the service: [end of service example](https://feedback-plugin-example.herokuapp.com/en/confirmation)

Since this PR is for end of service feedback, I'd like to just have the feedback link on the one page. Can you suggest a way to make it less prominent but only appear once the user has finished using the service?

**Comment by keithiopia** [comment]
@smcmurtry The end of service example you showed looks good. An equivalent on this transaction would be under the `[Put your agreement in an existing privacy form template to add the Canada Wordmark (opens in new tab).]` sentence.
Perhaps we can add another `hr` beneath that sentence, then put the link to the feedback component underneath. What do you think?

**Pull request closed without merging**